src/index.js

```diff
@@ -1,6 +1,25 @@
 import './style.css';
-import { scoreList, loadList } from './load.js';
+import { addScore, displayScores } from './api.js';
 
-const ulContain = document.querySelector('#score_list');
+const form = document.querySelector('#form');
+const refresh = document.getElementById('refresh_btn');
```

Comment:
- [ ] You are trying to access an element with the id `refresh_btn` button. but in your HTML file, it doesn't seem like there is such an element. this may be the reason why your website is not functioning correctly